config/webpack/webpack.config.base.js

```diff
@@ -132,6 +128,12 @@ module.exports = (env) => {
         APP_PATHS.ABS.SOURCE,
         APP_PATHS.ABS.NODE,
       ],
+      fallback: {
+        http: false,
+        zlib: false,
+        https: false,
+        stream: false
+      }
```

Comment:
I don't think this is necessary. I did not have to do this for the other apps.

Reply:
Without these,
<img width="904" alt="Screen Shot 2021-05-11 at 9 05 30 AM" src="https://user-images.githubusercontent.com/15723547/117766322-11b63a80-b238-11eb-8efe-d8fbab3b00ce.png">
 build fails:

Reply:
I'm quite certain you don't need the fallbacks. I'll take a look tomorrow.